client/src/components/overview/overview_components/SizeList.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+export default function Size({
+  skusArray,
+  skusObject,
+  selStyle,
+  setSelSize,
+  selSize,
+  totalStyleQuantity,
+  setSelSku
+}) {
+  const [optionsState, setOptionsState] = useState('Select Size');
+  // const [skuMap, setSkuMap] = useState({});
+  useEffect(() => {
+    setOptionsState('Select Size');
+  }, [selStyle]);
+  useEffect(() => {
```

Comment:
Why two useEffects? The dependency is the same so can't they be combined?

Reply:
I haven't refactored and the sizeList file is currently going through a lot of updates. It's something I noticed, but one of the dependencies I am working on is dependent on useEffect and I'm trying to fix functionality first.

Reply:
My intent is on a future/better full commit, I want to state "review all functionality with provided tests" so even if there are not any changes shown in git, it will be indication for a thorough review.

Reply:
Understood